guides/advanced_guides/tokenization.md

```diff
@@ -21,24 +21,26 @@ We can break down the tokenization process like so:
 
 ## Language Support
 
-Currently, the MeiliSearch tokenizer is optimized for four languages:
+Currently, MeiliSearch offers full support for two groups of languages:
 
-- **English**  🇬🇧
+- **English** 🇬🇧 and **any other language that uses the Latin alphabet**
 - **Chinese** 🇨🇳
-- **Japanese** 🇯🇵
-- **Korean** 🇰🇷
 
-It also supports any language that uses the Latin alphabet. **Results may vary in languages with long compound words, such as German.**
+This is due to how our [tokenizer](/guides/advanced_guides/tokenization.md#tokenization) is optimized. While we can't promise high-quality results for languages outside of these two groups, we find that **MeiliSearch works effectively in most languages**.
+
+Note that although we support all languages that use the Latin alphabet, **results may vary in languages with long compound words, such as German.**
 
 ## Future Development
 
-We rely on external contributors to help us extend our tokenizer to support new languages.
+While we have employees from all over the world at MeiliSearch, we don't speak every language. In fact, we rely almost entirely on external contributors to understand how our engine is performing in a given language.
 
-In the future, we hope that our international developer community will lead the process of developing additional tokenizer pipelines. For now, if there's an unsupported language that you would like us to target, you can help by writing tests for that language, giving us feedback, or simply giving a :+1: to the relevant GitHub issue.
+In the future, we plan to open-source the development of [tokenization pipelines](/guides/advanced_guides/tokenization.md#tokenization) and let our international community lead the charge towards a more global MeiliSearch. For now, you can help by sharing tests, results, and feedback with us in that language's GitHub issue. Be sure to also give a :+1: to any language you'd like us to target!
```

Comment:
I'm not sure about this. @MarinPostma could you confirm?

Reply:
There are no officials plan for the tokenizer, it's a long term effort, so d rather we do not commit on anything. You can point to the repo for now, so curious people can have a look and contribute is they want to.

Reply:
The tokenizer is open source, but the development is lead by us still 😊